iron-autogrow-textarea.html

```diff
@@ -78,7 +78,6 @@
         autocomplete$="[[autocomplete]]"
         autofocus$="[[autofocus]]"
         inputmode$="[[inputmode]]"
-        name$="[[name]]"
```

Comment:
Since `<iron-autogrow-textarea>` doesn't have form behavior, will it still work in a native form in shady dom or `iron-form` without the `name` attribute?